src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -805,13 +805,13 @@ object TezosDatabaseOperations extends LazyLogging {
       rawData.asCsvReader[A#TableElementType](rfc.withHeader.withCellSeparator(separator))
 
     // separates List[Either[L, R]] into List[L] and List[R]
-    val (errors, rows) = reader.toList.foldRight((List[ReadError](), List[A#TableElementType]()))(
-      (acc, pair) => acc.fold(l => (l :: pair._1, pair._2), rr => (pair._1, trimCaseClass(rr) :: pair._2))
+    val (errors, rows) = reader.toList.foldRight((List.empty[ReadError], List[A#TableElementType]()))(
```

Comment:
there's another List.empty, in the tuple 😛

Reply:
missed it 😄